aana/utils/json.py

```diff
@@ -35,11 +36,15 @@ def json_serializer_default(obj: object) -> object:
         return obj.model_dump()
     if isinstance(obj, Path):
         return str(obj)
+    if isinstance(obj, type):
+        return str(type)
+    if isinstance(obj, np.ndarray):
```

Comment:
ndarray should be handled by OPT_SERIALIZE_NUMPY in orjson. But if you use jsonify then it will not handle numpy. Can you check and see if we can unify `jsonify` and `orjson_serializer` and use orjson for everything?

Reply:
Added in [PR](https://github.com/mobiusml/aana_sdk/pull/154)